src/streamlink/plugin/api/useragents.py

```diff
@@ -22,3 +22,41 @@
 FIREFOX = "Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:52.0) Gecko/20100101 Firefox/52.0"
 FIREFOX_MAC = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10; rv:33.0) Gecko/20100101 Firefox/33.0"
 OPERA = "Opera/9.80 (Windows NT 6.0) Presto/2.12.388 Version/12.14"
+
+
+def random_agent():
+
+    import random
```

Comment:
Should import modules at the top, unless there is a specific reason not to?

Reply:
Lazy imports tend to work faster and since they are needed only locally in the function that why I put them there. This is only example however because the difference will be negligible anyway.